e2e/scenario_test/settings/not_set/basal_rates/settings.no_basal_rates.spec.js

```diff
@@ -0,0 +1,51 @@
+const { setup, pump, cgm, match, settings } = require('../../../../../src/index');
+
+describe('Pump Settings', () => {
+    beforeAll(async () => {
+        await setup.LaunchLoop();
+        await setup.LoadScenarios(device.id);
+        await cgm.Add();
+        await pump.Add();
+    });
+    describe('Closed loop is not allowed', () => {
+        describe('When basal rates are not set', () => {
+            beforeAll(async () => {
+                await setup.LoadScenario('flat_cgm');
+            });
+            it('should NOT set the suspend threshold', async () => {
+                await settings.Suspend('65');
+            });
+            it('should have correction range set', async () => {
+                await settings.CorrectionRanges([{time: '12:00 AM', min:'179',max:'180'}]);
+            });
+            it.skip('should set the basal rates', async () => {
+                await settings.BasalRates('0.1');
```

Comment:
[nit] Should this be an array of time and rate, much like the CorrectionRanges above?

Reply:
Also, the name of this test file is `no_basal_rates`, so should this be set here?

Reply:
1 - yes it should
2 - I skipped it `it.skip('should set the basal rates'` so it wasn't being set. Felt clever at the time but yeah ... :)